x-pack/plugins/stack_connectors/server/connector_types/pagerduty/index.ts

```diff
@@ -154,6 +154,7 @@ export function getConnectorType(): PagerDutyConnectorType {
       UptimeConnectorFeatureId,
       SecurityConnectorFeatureId,
     ],
+    canAutoRecover: true,
```

Comment:
Are you comfortable adding auto recovery to `pagerduty` before taking it for a spin through `slack` alerts?